lesson-3/task-1.js

```diff
@@ -0,0 +1,17 @@
+// 1. С помощью цикла while вывести все простые числа в промежутке от 0 до 100.
+
+
+var i = 0;
+while ( i <= 100) {
+    var is_prime = true;
```

Comment:
В JS принято использовать camelCase
```suggestion
    var isPrime = true;
```